.evergreen/config.yml

```diff
@@ -154,7 +157,25 @@ functions:
     - command: shell.exec
       params:
         script: |
```

Comment:
These evergreen changes are completely AI written except for the echo statement. Pretty impressive